test_portingdb/test_fixer_library.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+import subprocess
+import re
+import sys
+
+import pytest
+import modernize
+
+basepath = Path(__file__).parent.parent / 'source'
+all_rst_files = [str(p.relative_to(basepath))
+                 for p in basepath.glob('**/*.rst')]
+
+# Fixer names are in the format "<module>.fixes.fix_<name>"
+FIXER_RE = re.compile(r'\w+\.fixes\.fix_\w+')
+
+
+@pytest.mark.parametrize('filename', all_rst_files)
+def test_fixers_exist(filename, fixer_names):
+    """Test that all mentioned fixers are correctly named and available"""
+    path = basepath / filename
+    with path.open() as f:
+        for lineno, line in enumerate(f, start=1):
+            for match in FIXER_RE.finditer(line):
+                found_text = match.group(0)
+                print(f'{filename}:{lineno}:{line.strip()}')
+                assert found_text in fixer_names
+
+
+@pytest.fixture(scope='module')
```

Comment:
I usually put fixtures above tests, as the reader has to understand the fixture first before reading the test.